src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaFunctionSelect.tsx

```diff
@@ -92,7 +106,27 @@ export default class AwsLambdaFunctionSelect extends React.Component<Props, Stat
 
   renderCore = () => {
     const {initialStepNumber} = this.props;
+    const {toggleAllState} = this.state;
     const model = this.model;
+
+    const toggleHelp = toggleAllState ? t('Disable All') : t('Enable All');
+
+    const FormHeader = (
+      <StyledPanelHeader>
+        {t('Lambda Functions')}
+        <SwitchHolder>
+          <Tooltip title={toggleHelp} position="left">
+            <StyledSwitch
+              size="lg"
+              name="toggleAll"
+              toggle={this.handleToggle}
+              isActive={toggleAllState}
+            />
+          </Tooltip>
+        </SwitchHolder>
+      </StyledPanelHeader>
```

Comment:
I think this would become and observe and it would observe the state of all checkboxes in the modal